app/models/mixins/reportable_mixin.rb

```diff
@@ -150,23 +150,19 @@ def add_includes(data_records, options)
       end
 
       if association == "categories"
-        association_objects = []
-        assochash = {}
+        association_hash = {}
         includes["categories"][:only].each do|c|
           entries = Classification.all_cat_entries(c, self)
           entarr = []
           entries.each { |e| entarr.push(e.description) }
```

Comment:
May I use `map` instead of `collect` there? I generally see `map` being used much more often than `collect` in Ruby world.

Not that it mattered too much, since it is going to be replaced with `pluck` anyway (I'm working on another PR in order to be able to do this), but anyway: what is your preference, map or collect?